spells.yml

```diff
@@ -8914,11 +8914,11 @@ shadow_blade:
   components: V, S
   concentration: true
   description: |-
-    You weave together threads of shadow to create a sword of solidified gloom in your hand. This magic sword lasts until the spell ends. It counts as a simple melee weapon with which you are proficient. It deals 2d8 psychic damage on a hit and has the finesse, light, and thrown properties (range 20/60). In addition, when you use the sword to attack a target that is in dim light or darkness, you make the attack roll with advantage.
+    You weave together threads of shadow to create a sword of solidified gloom in your hand. This magic sword lasts until the spell ends. It counts as a simple melee weapon with which you are proficient. It deals 1d8 psychic damage on a hit and has the finesse, light, and thrown properties (range 20/60). In addition, when you use the sword to attack a target that is in dim light or darkness, you make the attack roll with advantage.
 
     If you drop the weapon or throw it, it dissipates at the end of the turn. Thereafter, while the spell persists, you can use a minor action to cause the sword to reappear in your hand.
   duration: Up to 1 minute
-  higher_level: When you cast this spell using a 3rd- or 4th-level spell slot, the damage increases to 3d8. When you cast it using a 5th- or 6th-level spell slot, the damage increases to 4d8. When you cast it using a spell slot of 7th level or higher, the damage increases to 5d8.
+  higher_level: When you cast this spell using a spell slot of 3rd level or higher, the damage increases by 1d8 for every two slot levels above 2nd.
```

Comment:
Shouldn't it be `of 4th level or higher`?

Reply:
This wording is actually identical to 5e's spiritual weapon, FWIW.

Reply:
I suppose it can make sense